src/applications/vaos/tests/e2e/appointment-list.cypress.spec.js

```diff
@@ -258,6 +258,7 @@ describe('VAOS appointment list refresh', () => {
       cy.get('[data-cy=appointment-list-item]')
         .first()
         .findByText(/Details/i)
+        .focus()
```

Comment:
Not sure why this test started failing, but focus on the link before clicking fixed it (as it does for other weird cypress issues).